<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="SPTVMLAP-2029" failures="0" tests="2" name="Core.SecurityManagement.BannedCustomerTest" time="-1737352192.000" errors="1" timestamp="2025-01-20T11:30:29 IST" skipped="0">
  <testcase classname="Core.SecurityManagement.BannedCustomerTest" name="takeScreenshotOnFailure" time="-1737352832.000">
    <error message="
Can inject only one of &lt;ITestContext, XmlTest&gt; into a @AfterSuite annotated takeScreenshotOnFailure.
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
Can inject only one of <ITestContext, XmlTest> into a @AfterSuite annotated takeScreenshotOnFailure.
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection
at org.testng.internal.Parameters.checkParameterTypes(Parameters.java:439)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:353)
at org.testng.internal.Parameters.createParameters(Parameters.java:708)
at org.testng.internal.Parameters.createConfigurationParameters(Parameters.java:168)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:302)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:344)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- takeScreenshotOnFailure -->
  <system-out/>
  <testcase classname="Core.SecurityManagement.BannedCustomerTest" name="AccessBannedCustomerPage" time="713.711"/>
  <system-out/>
</testsuite> <!-- Core.SecurityManagement.BannedCustomerTest -->
